ReleaseNotes.md

```diff
@@ -11,6 +11,10 @@ GitHub)
 
 ## Backwards incompatible changes
 
+HasPropWithValueQueryBase used RDKit::Dict::Pair to return
```

Comment:
This should be a bulleted list, please start the first line with `- ` like is in the other sections of this document